js/components/closed-caption-multi-audio-menu/helpers.js

```diff
@@ -41,9 +42,8 @@ function getDisplayLanguage(languagesList, languageCode) {
   ) {
     var matchingLanguage = _.find(languagesList, function(language) {
       return (
-        language['1'] === languageCode ||
-        language['2'] === languageCode ||
         language['2T'] === languageCode ||
+        language['1'] === languageCode ||
```

Comment:
Just to double check, do we no longer care about language['2']?

Reply:
I found a table on [wiki](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) . It looks like the most full table with codes. I have grabbered it. So now in our map (language.js file) we have all single values for ISO-639-2 from first column and values with asterisk from first column , they are marked as "2B", and values without asterisk are marked as "2T".

About language['x']:
There is keys "1", "2B", "2T", "3" in languages.js . These are types of ISO-639 .
You can read about the types here: 
https://en.wikipedia.org/wiki/ISO_639-1
https://en.wikipedia.org/wiki/ISO_639-2
https://en.wikipedia.org/wiki/ISO_639-3

And we check if the langCode is one of these codes.

Reply:
Thanks for the great explanation @KseniiaSokolovaEpam !